openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -600,12 +600,18 @@ grant_type=urn:ietf:params:oauth:grant-type:pre-authorized_code
 
 Token Responses are made as defined in [@!RFC6749].
 
+The AS might decide to authorize issuance of multiple instances for each Credential requested in the Authorization Request. Each Credential instance is described using the same entry in the `credentials_supported` Credential Issuer metadata, but contains different claim values or different subset of claims within the claimset identified by the Credential type.
```

Comment:
Maybe it would be good to find a different word instead of "credential type" to describe an entry in the `credentials_supported` Credential Issuer metadata. The terms feels too overloaded.

How about "credential description" or "credential definition"?

Reply:
great suggestion! thank you, Daniel.

```suggestion
The AS might decide to authorize issuance of multiple instances for each Credential requested in the Authorization Request. Each Credential instance is described using the same entry in the `credentials_supported` Credential Issuer metadata, but contains different claim values or different subset of claims within the claimset identified by the Credential description.
```